projects/core/src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -824,7 +844,13 @@ unserialise = unserialize -- GB version
 --
 -- @param t The value to serialise. Like @{textutils.serialise}, this should not
 -- contain recursive tables or functions.
--- @tparam[opt] boolean bNBTStyle Whether to produce NBT-style JSON (non-quoted keys)
+-- @tparam[opt] { nbt_style? = boolean, unicode_strings? = boolean } options Options for serialization.
+--  - `nbt_style`: Whether to produce NBT-style JSON (non-quoted keys) instead of standard JSON.
+--  - `unicode_strings`: Whether to parse strings as containing UTF-8 characters instead of
+--    using the default 8-bit ANSI character set.
```

Comment:
I think it's probably a little confusing to describe this as parsing (it is, but probably best reserved to the parsing `unserialiseJSON` does).

```suggestion
--  - `unicode_strings`: Whether to treat strings as containing UTF-8 characters instead of
--    using the default 8-bit character set.
```